script.js

```diff
@@ -0,0 +1,841 @@
+const FUNCTIONAL_KEYS = {

```

Comment:
Move constants to another file.

Reply:
The state of functional keys was moved to class property.

https://github.com/BlackTruth/virtual-keyboard/blob/6297a0817c03affb1948c9e73a6352687e6ff449/src/classes/VirtualKeyboard.js#L7-L20